core/API/CORSHandler.php

```diff
@@ -26,23 +26,24 @@ public function __construct()
     public function handle()
     {
         if (empty($this->domains)) {
-            return;
+            Common::sendHeader('Access-Control-Allow-Origin: *');
```

Comment:
do we really want to allow all origins if the "cors_domains" setting is empty?

if yes, we never should return `*` to the browser if there is a HTTP_ORIGIN set. returning  `*` is deprecated and some browsers will ignore it.

we would need something like this:
```
            if (!empty($_SERVER['HTTP_ORIGIN'])) {
                Common::sendHeader('Access-Control-Allow-Origin: ' . $_SERVER['HTTP_ORIGIN']);
                return false;
            }

            Common::sendHeader('Access-Control-Allow-Origin: *');
            return false;
```